apps/web/.env.test

```
@@ -0,0 +1,19 @@
+NODE_ENV=test
+DATABASE_URL=postgresql://postgres:password@localhost:5432/inboxzero?schema=public
+DIRECT_URL=postgresql://postgres:password@localhost:5432/inboxzero?schema=public
+NEXT_PUBLIC_BASE_URL=http://localhost:3000
+GOOGLE_CLIENT_ID=test-google-client-id
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The GOOGLE_CLIENT_ID key should go before the NEXT_PUBLIC_BASE_URL key

(UnorderedKey)